### ISSUE TITLE
Generates duplicates List of objects

### ISSUE THREAD

**Issue by nbelley** [Virtuoze/Swagger4WCF#11]
It seems the program generates weird duplicate list of objects:

    definitions:
      List`1:
        type: object
        properties:
          Capacity:
            type: "number"
            format: int32
          Count:
            type: "number"
            format: int32
          System.Collections.IList.IsFixedSize:
            type:" boolean"
          System.Collections.Generic.ICollection<T>.IsReadOnly:
            type:" boolean"
          System.Collections.IList.IsReadOnly:
            type:" boolean"
          System.Collections.ICollection.IsSynchronized:
            type:" boolean"
          System.Collections.ICollection.SyncRoot:
            $ref: "#/definitions/Object"
          Item:
            $ref: "#/definitions/T"
          System.Collections.IList.Item:
            $ref: "#/definitions/Object"
      List`1:
        type: object
        properties:
          Capacity:
            type: "number"
            format: int32
          Count:
            type: "number"
            format: int32
          System.Collections.IList.IsFixedSize:
            type:" boolean"
          System.Collections.Generic.ICollection<T>.IsReadOnly:
            type:" boolean"
          System.Collections.IList.IsReadOnly:
            type:" boolean"
          System.Collections.ICollection.IsSynchronized:
            type:" boolean"
          System.Collections.ICollection.SyncRoot:
            $ref: "#/definitions/Object"
          Item:
            $ref: "#/definitions/T"
          System.Collections.IList.Item:
            $ref: "#/definitions/Object"
      List`1:
        type: object
        properties:
          Capacity:
            type: "number"
            format: int32
          Count:
            type: "number"
            format: int32
          System.Collections.IList.IsFixedSize:
            type:" boolean"
          System.Collections.Generic.ICollection<T>.IsReadOnly:
            type:" boolean"
          System.Collections.IList.IsReadOnly:
            type:" boolean"
          System.Collections.ICollection.IsSynchronized:
            type:" boolean"
          System.Collections.ICollection.SyncRoot:
            $ref: "#/definitions/Object"
          Item:
            $ref: "#/definitions/T"
          System.Collections.IList.Item:
            $ref: "#/definitions/Object"
      List`1:
        type: object
        properties:
          Capacity:
            type: "number"
            format: int32
          Count:
            type: "number"
            format: int32
          System.Collections.IList.IsFixedSize:
            type:" boolean"
          System.Collections.Generic.ICollection<T>.IsReadOnly:
            type:" boolean"
          System.Collections.IList.IsReadOnly:
            type:" boolean"
          System.Collections.ICollection.IsSynchronized:
            type:" boolean"
          System.Collections.ICollection.SyncRoot:
            $ref: "#/definitions/Object"
          Item:
            $ref: "#/definitions/T"
          System.Collections.IList.Item:
            $ref: "#/definitions/Object"

**Comment by DelvalletQuentin** [comment]
Yes, I have the same problem, it fails to generate the documentation for collections.

I do think it's the case with every generic types.
